e2e/workflow/installation.go

```diff
@@ -117,6 +117,10 @@ func (w *InstallationSuite) GetCI(ctx context.Context) error {
 	return nil
 }
 
+func (w *InstallationSuite) CheckClusterInstallationStatus(ctx context.Context) error {
+	return pkg.WaitForClusterInstallationReadyStatus(w.client, w.Meta.ClusterInstallationID, w.logger)
```

Comment:
Maybe we can merge `GetCI` and `CheckClusterInstallationStatus` into the  same step?

Reply:
In that case, do we need a better name than `GetCI`?

Reply:
I'm leaving it now as it it.